docs/release/upgrade.adoc

```diff
@@ -46,6 +46,9 @@ We generally always recommend updating your apps to the latest version!
 
 == v7.9 notes
 
+* <<deployment/config#_portal, Portal config>> now specifies default Content-Security-Policy for page `preview`/`inline`/`edit` mode (it does not affect `live` mode) .
+It is possible to turn the header off or configure via `page.defaultContentSecurityPolicy` value.
```

Comment:
It is possible to configure the header via `page.defaultContentSecurityPolicy` value or go back to previous behaviour by setting `page.defaultContentSecurityPolicy` to an empty value.